src/main/java/net/fabricmc/loom/configuration/ide/RunConfigSettings.java

```diff
@@ -310,6 +331,31 @@ public void server() {
 	public void data() {
 		environment("data");
 		defaultMainClass(getExtension().isForge() ? Constants.Forge.LAUNCH_TESTING : Constants.Knot.KNOT_SERVER);
+
+		if (getExtension().isForge()) {
+			forgeTemplate("data");
+		}
+	}
+
+	/**
+	 * Applies a Forge run config template to these settings.
+	 *
+	 * @param templateName the template name (usually one of {@code server}, {@code client}, {@code data})
+	 * @since 1.0
+	 */
+	public void forgeTemplate(String templateName) {
+		ModPlatform.assertPlatform(getExtension(), ModPlatform.FORGE);
+		// Evaluate later if Forge hasn't been resolved yet.
+		evaluateNowOrLater(() -> {
```

Comment:
Won't this cause issue if you try to override this? It might be better to move this to the constructor and store a forgeTemplateName variable in this settings.

Reply:
Fyi the templates overwrite user config in 0.12 too, but yeah that's probably better